tests/test_hstrat/test_stratum_retention_strategy/test_stratum_retention_algorithms/test_depth_proportional_resolution_algo/test_PolicySpec.py

```diff
@@ -3,6 +3,7 @@
 
 import pytest
 
+from hstrat import hstrat
```

Comment:
move this into body of function right above eval

Reply:
Fixed in 0f0e2f2.